modules/application-health-about.adoc

```diff
@@ -5,125 +5,215 @@
-36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulling     pulling image "k8s.gcr.io/busybox"
-36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Pulled      Successfully pulled image "k8s.gcr.io/busybox"
-36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Created     Created container with docker id 86849c15382e; Security:[seccomp=unconfined]
-36s       36s     1   {kubelet worker0}   spec.containers{liveness}   Normal      Started     Started container with docker id 86849c15382e
-2s        2s      1   {kubelet worker0}   spec.containers{liveness}   Warning     Unhealthy   Liveness probe failed: cat: can't open '/tmp/healthy': No such file or directory
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    test: health-check
+  name: my-application
+...
+spec:
+  containers:
+  - name: goproxy-app <1>
+    args:
+    image: k8s.gcr.io/goproxy:0.1 <2>
+    livenessProbe: <3>
+      exec: <4>
+        command: <5>
+        - /bin/bash
+        - '-c'
+        - timeout 60 /opt/eap/bin/livenessProbe.sh 
+      periodSeconds: 10 <6>
+      successThreshold: 1 <7>
+      failureThreshold: 3 <8>
+...
 ----
 
-Startup Probe::
-Legacy applications can require additional startup time on their first initialization. This situation can make it difficult to set up liveness probe parameters without compromising the fast response to deadlocks that a liveness probe provides. To prevent containers from starting slowly, configure a startup probe using the same command, HTTP or TCP check, with a `failureThreshold * periodSeconds` value long enough to handle the worst case startup time.
+<1> The container name.
+<2> Specify the container image to deploy.
+<3> The Liveness probe.
+<4> The type of probe, here a Container Command probe.
+<5> The commands to execute on the container.
+<6> How often in seconds to perform the probe. 
+<7> The number of number of consecutive successes needed to show success after a failure.
+<8> The number of times to try the probe after a failure.
 
-For example, add the startup probe to the to the previous liveness check sample using a maximum of 5 minutes (30 * 10 = 300s). After the startup probe succeeds the first time, the liveness probe takes over to provide a fast response to container deadlocks. If the startup probe never succeeds within the specified time period, the container is killed and is subject to the pod's `restartPolicy`.
-
-
-.Sample Startup Check
-[source, yaml]
+.Sample Readiness probe and Liveness probe with a TCP Socket test in a deployment
+[source,yaml]
 ----
- startupProbe:
-   httpGet:
-     path: /healthz
-     port: liveness-port <1>
-   failureThreshold: 30 <2>
-   periodSeconds: 10 <3>
+kind: Deployment
+apiVersion: apps/v1
+....
+spec:
+....
+    spec:
+      containers:
+        - resources: {}
```

Comment:
So this is confusing. I think it's because you're doing

```
spec:
... all this is missing
    spec:
```

But depth 2 probably has `template:` or some such.

I think every parent level must manifest in example output, or it isn't clear where you are.

It looks like it's `template` though.

```
kind: Deployment
apiVersion: apps/v1
spec:
  template:
    spec:
```

https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/deployment-apps-v1.html#deployment-apps-v1